instrumentation/opentelemetry-instrumentation-tornado/src/opentelemetry/instrumentation/tornado/__init__.py

```diff
@@ -0,0 +1,263 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This library uses OpenTelemetry to track web requests in Tornado applications.
+
+Usage
+-----
+
+.. code-block:: python
+
+    import tornado.web
+    from opentelemetry.instrumentation.tornado import TornadoInstrumentor
+
+    # apply tornado auto-instrumentation
```

Comment:
I don't think we should say "auto-instrumentation" since we usually use this to refer to not having to instrument manually.